src/api/followers.py

```diff
@@ -0,0 +1,84 @@
+from sqlalchemy.ext.asyncio import AsyncSession
+from fastapi import APIRouter, Depends, status, HTTPException
+
+from src.domain.page_info import PageInfo
+from src.database.database_config import get_db
+from src.database.models.user import UserDatabaseHandler
+from src.database.models.user_tag import UserTagDatabaseHandler
+from src.domain.user_tag import PaginatedUserTagAlcohols, PaginatedUserTag
+
+router = APIRouter(prefix='/followers', tags=['followers'])
```

Comment:
Może przenieść to do /users. 
Followers sugeruje, że trzeba kogoś obserwować, żeby to pobierać